src/coreclr/inc/corhlprpriv.h

```diff
@@ -11,7 +11,7 @@
 #define __CORHLPRPRIV_H__
 
 #include "corhlpr.h"
-#include "fstring.h"
+#include "minipal/utf8.h"
```

Comment:
nit: for consistency, use angular brackets.
```suggestion
#include <minipal/utf8.h>
```